src/Parser/Shaman/Elemental/Modules/Talents/ExposedElements.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import enemy from 'Parser/Core/Entity';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import { formatNumber, formatPercentage } from 'common/format';
+import SpellIcon from 'common/SpellIcon';
+
+import SPELLS from 'common/SPELLS/SHAMAN';
+
+import { EXPOSED_ELEMENTS } from 'Parser/Shaman/Elemental/Constants';
+
+class ExposedElements extends Analyzer {
+  removeDebuffTimestamp = null;
+  damageGained=0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasTalent(SPELLS.EXPOSED_ELEMENTS_TALENT.id);
+  }
+
+  on_byPlayer_cast(event){
+    if(this.removeDebuffTimestamp === null)
+      return;
+
+    if((event.ability.guid === SPELLS.LIGHTNING_BOLT.id) && (enemy.hasBuff(SPELLS.EXPOSED_ELEMENTS_TALENT.id))){
+      this.removeDebuffTimestamp=event.timestamp;
+      this.damageGained+=(1/EXPOSED_ELEMENTS.MULTIPLIER)*event.amount;
```

Comment:
please use the helper for this:
https://github.com/WoWAnalyzer/WoWAnalyzer/blob/c81403a4dba387b3e984642cbf14fda4b9913228/src/Parser/Core/calculateEffectiveDamage.js#L1
(note that it takes increase and not multiplier as a parameter, so a 100% increase would just be `1`)

Reply:
Didn't know that one existed. I just changed the const to work like this now.